protocol/tx/spend.go

```diff
@@ -9,17 +9,43 @@ type spend struct {
 		ExtHash     bc.Hash
 	}
 	ordinal int
+
+	// SpentOutput contains (a pointer to) the manifested entry
+	// corresponding to body.SpentOutput. It may be nil, in which case
+	// spentInfo must be non-nil.
+	SpentOutput *output
+
+	// spentInfo contains validation-essential data in case the spent
+	// output object is not present (in SpentOutput).
```

Comment:
I am pretty sure we need the previous output in the transaciton. Even aside from the party who's spending the money, any validating node needs a copy of the prevout in order to validate the tx. In txgraph-land, we've elminiated the "Output Commitment" and replaced with a full "output" entry. I don't see any way for a validator to do its job using just the synthesized spentInfo. How would it get the data to put in there?

@danrobinson?